File: MediCareWebSpa/src/app/DTOs/request/feedback-request.dto.ts
export class FeedbackRequestDTO {
  rate: number;
  description: string;
  appointmentId: number;

  constructor(
    rate: number,
    description: string,
    appointmentId: number
  ) {
    this.rate = rate;
    this.description = description;
    this.appointmentId = appointmentId;
  }
}

File: MediCareWebApi/DTOs/Request/FeedbackRequestDTO.cs
using System.ComponentModel.DataAnnotations;

namespace MediCare.DTOs.Request
{
	public class FeedbackRequestDTO
	{
		[Required(ErrorMessage = "Ocena jest wymagana.")]
		[Range(1, 5, ErrorMessage = "Ocena musi być między 1 a 5.")]
		public byte Rate { get; set; }

		[Required(ErrorMessage = "Opis jest wymagany.")]
		[MaxLength(256, ErrorMessage = "Opis nie może przekroczyć 256 znaków.")]
		public string Description { get; set; }

		[Required(ErrorMessage = "Id wizyty jest wymagane.")]
		public int AppointmentId { get; set; }
	}
}
